GameData/RP-0/changelog.cfg

```
@@ -10,16 +10,15 @@ KERBALCHANGELOG
 		versionKSP = 1.12.3
 		CHANGE
 		{
-			change = Switched to KSP-RO
+			change = Switched to KSP-RO
 			subchange = now uses KSP-RO
 		}
 	}
 	VERSION
 	{
 		version = 0.0.0.7
 		versionKSP = 1.12.3
-		CHANGE
-		{
+		CHANGE{
```

Here is a suggestion
```suggestion
		CHANGE
		{
```